### ISSUE TITLE
warning: unknown conversion type character ‘R’ in format [-Wformat=]

### ISSUE THREAD

**Issue by mgorny** [dottedmag/pychm#6]
When building with Python 2.7, I get:

```
chm/_chmlib.c: In function ‘chmlib_chm_enumerator’:
chm/_chmlib.c:138:18: warning: unknown conversion type character ‘R’ in format [-Wformat=]
  138 |                  "chm_enumerate callback is expected to return "
      |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
chm/_chmlib.c:139:46: note: format string is defined here
  139 |                  "integer or None, returned %R",
      |                                              ^
chm/_chmlib.c: In function ‘chmlib_chm_enumerate_dir’:
chm/_chmlib.c:175:78: warning: unknown conversion type character ‘R’ in format [-Wformat=]
  175 |     PyErr_Format(PyExc_TypeError, "A callable is expected for callback, got %R",
      |                                                                              ^
chm/_chmlib.c: In function ‘chmlib_chm_enumerate’:
chm/_chmlib.c:210:78: warning: unknown conversion type character ‘R’ in format [-Wformat=]
  210 |     PyErr_Format(PyExc_TypeError, "A callable is expected for callback, got %R",
      |                                                                              ^
chm/_chmlib.c: In function ‘chmlib_search’:
chm/_chmlib.c:340:78: warning: unknown conversion type character ‘R’ in format [-Wformat=]
  340 |     PyErr_Format(PyExc_TypeError, "A callable is expected for callback, got %R",
      |                                                                              ^
```

This doesn't happen with newer Python versions, so I suspect you might be relying on some new Python 3 feature.

**Comment by dottedmag** [comment]
You're right. Compatibility restored.

**Comment by dottedmag** [comment]
Released in 0.8.6
